boards/nrf51dk/src/main.rs

```diff
@@ -261,15 +261,35 @@ pub unsafe fn reset_handler() {
 
 
 use core::fmt::Arguments;
+use core::fmt::Write;
+use core::fmt::write;
 #[cfg(not(test))]
 #[lang="panic_fmt"]
 #[no_mangle]
-pub unsafe extern "C" fn rust_begin_unwind(_args: &Arguments,
+pub unsafe extern "C" fn rust_begin_unwind(_args: Arguments,
```

Comment:
Actually, I believe that `#[lang="panic_fmt"]` changes the function name to `rust_begin_unwind` because our crates set `panic=abort`, and on a panic rust's behavior is to begin unwinding, thus begin unwind is the function that's called.

I'd just leave the function name alone. The magic name-changing is under debate/flux in rust anyway.